examples/pub/lib/pub_repository.dart

```diff
@@ -7,15 +7,21 @@ part 'pub_repository.g.dart';
 
 class PubRepository {
   final dio = Dio();
+  final scheme="https";
+  final host='pub.dartlang.org';
```

Comment:
Could you make those static private consts?

Also, this needs formatting and should use single quotes.

Reply:
ok

Reply:
I did it